docs/en/your-profile-and-preferences/customise-mentions-and-other-messages.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 570
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Customise mentions and other messages
+
+You can customise three different types of notifications: `Personal mentions`, `Global Mentions` and `All other messages`. For each one of them, follow the same procedure and use the table below as a reference.
+
+## Know your notification options
+
+| Option | Description | Works with |
+|:---|:---|:---|
+| Personal metions| Include messages that specifically mention you, or have your name tagged. | [Mobile and desktop] |
+| Global mentions | Include notifications for announcements or messages that are directed to the entire community. | [Mobile and desktop] |
+| All other messages| Include any messages that don't use the "personal mentions" or "global mentions".| [Mobile and desktop]
+| Same settings for all message types| Include [add a definition here]". | [Mobile and desktop] |
+
+!!! note
+	These notifications apply to communities, channels, group chats and direct messages (DMs).
+
+## Customise mentions and all other messages
+
+You can choose to be notified about mentions (personal and global) or every new message (all other messages).
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-communities: **Communities** to customise a community or channel, or tap :mobile-messages: **Messages** to customise a group chat or DM.
```

Comment:
- 'customize'
- 'customize your notification settings for' or 'set up notifications for'?